client/components/questionnaire1.js

```diff
@@ -53,18 +89,25 @@ class Questionnaire1 extends Component {
         <form className="questionnaire-container" onSubmit={this.handleSubmit}>
           {questionnairePage1.map(current => {
             return (
-              <p key={current.questionNumber}>
+              <div key={current.questionNumber}>
                 <label htmlFor={current.name}>
                   Question {current.questionNumber}: {current.title}*
                 </label>
                 <input
                   name={current.name}
-                  type="number"
+                  type="string"
```

Comment:
Would changing the input to a number work?

Reply:
It technically does, but when all the fields are initialized with 0 then it doesn't notify the user if they didn't fill out that field. When it's a string, they have to put zero if it doesn't apply.